cuchem/benchmark/scripts/benchmark.yaml

```diff
@@ -5,31 +5,34 @@ model:
 
 metric:
   validity:
-    enabled: True
+    enabled: False
     radius:
       - 1
       - 2
 
   unique:
-    enabled: false
+    enabled: False
     radius: ${metric.validity.radius}
 
   novelty:
     enabled: False
     radius: ${metric.validity.radius}
 
   nearestNeighborCorrelation:
```

Comment:
Personal preference, but I'd prefer no camel case and use underscores here -- 'nearest_neighbor_correlation'. Seems more inline with yaml style